src/plopp/backends/backend_manager.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
A general comment on the class. Simon had found this article: https://treyhunner.com/2019/04/why-you-shouldnt-inherit-from-list-and-dict-in-python/

I think it would be better to inherit from `MutableMapping`.

Also, `__setitem__` should check if the backend name is valid.